index.html

```diff
@@ -26,6 +26,159 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section>
+            <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
+            <h1 class="recent-works">My Recent Works</h1>
```

Comment:
It's important to follow semantics and best practices to improve our projects as much as we can! The h1 is used as the main title of a website because represents the main heading/subject for the whole page so just used it once throughout your website😉 You could use from h2 to h6 tags in a hierarchical manner with the rest of your texts 📝Please change all the `h1` tags in your project so that you only remain with 1